projects/lyonel_feininger/index.md

```diff
@@ -22,20 +22,20 @@ The project was developed for the class “Gestalten in Code – Die Grundlagen
 Due to his works at Bauhaus 1919, Lyonel Feininger is one of the most eminent artist of the period of the “Classical Modernity”. 
 Lyonel Feininger was born 1871 in New York City. When he came to Europe 1887 he studied in Hamburg, Berlin and then Paris, France. 1919 he was called to Bauhaus University by Walter Gropius to be the first school master for painting and graphics. 
 
-![Lyonel-Feininger](assets/001_lyonel_feininger.png)
+![Lyonel-Feininger](assets/images/001_lyonel_feininger.png)
```

Comment:
the paths to the files are still not relativ it has to be   

    ![](./assets/images/img.png)  

the `./` before assets is important